core/modules/installer/css/installer.css

```diff
@@ -0,0 +1,251 @@
+#installer-browser-install-button-form {
+  clear: both;
+  margin: 10px 0 0;
+}
+
+#installer-browser-install-queue {
+  margin: 5px 0 0;
+}
```

Comment:
It's preferable to use classes if possible in our CSS to avoid specificity wars.